Macaulay2 1.25.11

d-torrance opened the pull request:
Target date: November 15, 2025

TODO:
- [x] Various pull requests
- [x] Add certified packages from JSAG
- [x] Update changelog (TODO: entries for any pending PR's)
- [x] Update syntax highlighting files and M2-emacs module (wait until any PR's that introduce new symbols are merged first)

d-torrance committed:
Bump version number to 1.25.11

d-torrance committed:
Certify ExteriorExtensions, MatrixSchubert, & MultigradedBGG

Three most recent JSAG-published packages

d-torrance committed:
Add changelog for 1.25.11

d-torrance committed:
Build ARM RPM's too

d-torrance committed:
Don't build Fedora 40 RPM (EOL on 2025-05-13)

d-torrance committed:
Bump upload-artifact action to v5 in build-rpm workflow

d-torrance committed:
Bump KaTeX to 0.16.25

d-torrance committed:
Remove submodule "trick" code from RHEL rpm builds

The autotools build now knows how to handle submodule builds when
building outside a git repository.

d-torrance committed:
Stop building "fat" tarball w/ submodules for rpm builds

No longer necessary since we download the submodules during the build
now.

mahrud commented on M2/Macaulay2/packages/Macaulay2Doc/changes.m2:
```diff
@@ -66,36 +67,62 @@ changesHelper List := opt -> pkgnames -> (
 	LI { BOLD "upcoming breaking changes:",
 	    UL {
-		LI {}
+		LI { "Beginning in the 1.26.05 release, current routines involving chain complexes and graded modules
```
Might want to remove this warning from the "changes, 1.25.05" node.

d-torrance committed:
Remove Complexes warning from 1.25.05 changelog entry

No longer accurate since Complexes is currently planned for the
1.26.05 release.

d-torrance committed:
Add url for downloading Macaulay2-docs "library"

d-torrance committed:
Version bumps for several of my packages

d-torrance committed:
Deal w/ LieTypes -> LieAlgebraRepresentation renaming

* Move installation back where it was, before ConformalBlocks, which
  depends on it
* Re-certify (w/ legacy name)
* needsPackage "LieTypes" still works (w/ warning)

d-torrance committed:
Switch Divisors -> WeilDivisors in NonPrincipalTestIdeals exports

This avoids package renaming warning

d-torrance committed:
Fix Visualize bug (missing initialization of clipboard for graphs)

d-torrance committed:
Add lcm & 1-arg gcd for Python < 3.9

d-torrance committed:
Update M2-emacs commit w/ 1.25.11 symbols

d-torrance committed:
Update prism.js symbols for version 1.25.11.

d-torrance committed:
Update vim symbols for 1.25.11

d-torrance committed:
Don't fail because of texinfo installation warning

d-torrance committed:
Restore support for Python < 3.8 (when PyStatus was introduced)
